### PR TITLE
[SP-6256] coralogix steps updating to new standards

### PR DESCRIPTION
**Pull request by igorkheif**

### CODE REVIEW

File: steps/coralogix/base/base.go [hunk 1]
```diff
@@ -1,5 +1,100 @@
...
+	case httpCode/100 == 2:
+		retStatus = status.OK
+	case httpCode == http.StatusNotFound:
+		retStatus = status.New(codes.NotFound, "resource not found", "")
+	case httpCode == http.StatusUnauthorized:
+		retStatus = status.New(codes.Unauthenticated, "authentication failed, token parameters", "")
+	case httpCode/100 == 4:
+		retStatus = status.New(codes.InvalidArgument, "encountered a client side error, please check all parameters and try again", "")
+	case httpCode/100 == 5:
+		retStatus = status.New(codes.Internal, "coralogix has encountered an internal error", "")
```
Comment (hagaishapira) [inline steps/coralogix/base/base.go:94]: Not sure this is code internal, the error is not internal to us but the API is unavailable.

File: steps/coralogix/base/base.go [hunk 2]
```diff
@@ -1,5 +1,100 @@
...
+	default:
+		retStatus = status.New(codes.Internal, "encountered an undocumented error", "")
```
Comment (hagaishapira) [inline steps/coralogix/base/base.go:96]: again this is not an "internal" error to the step's code

File: steps/coralogix/query/manifest.yaml [hunk 3]
```diff
@@ -28,3 +32,75 @@ envs:
...
+    description: "Contains a count of shards used for the request."
+    type: json
+    example: |
+      {
+          "failed": 0,
+          "skipped": 0,
+          "successful": 6,
+          "total": 6
+      }
+  - name: hits
```
Comment (hagaishapira) [inline steps/coralogix/query/manifest.yaml:56]: let's put this first

**Review by hagaishapira — APPROVED** [review]
See small comments, but overall looks good. Merge with @oriser's review

File: steps/coralogix/base/base.go (hunk 2 above)
Comment (igorkheif) [reply steps/coralogix/base/base.go:96]: What should this one be? Unavailable as well?

File: steps/coralogix/base/base.go (hunk 1 above)
Comment (igorkheif) [reply steps/coralogix/base/base.go:94]: Talked about it, should talk later on how we address statuses, for now leaving as is

File: steps/coralogix/base/base.go (hunk 2 above)
Comment (igorkheif) [reply steps/coralogix/base/base.go:96]: Talked about it, should talk later on how we address statuses, for now leaving as is

File: steps/coralogix/query/manifest.yaml (hunk 3 above)
Comment (igorkheif) [reply steps/coralogix/query/manifest.yaml:37]: Done

**Review by oriser — APPROVED** [review]
LGTM

**Pull request merged**